_data/providers.json

```diff
@@ -6,6 +6,11 @@
     "java_ip": "Make sure your remote address is 'auto', uncomment `bedrock address`, and change `0.0.0.0` to your Java server's IP."
   },
   "built_in": [
+    {
+      "name": "GameProHost",
```

Comment:
The list is made to be alphabetical, and is divided into three sections - Built-in Geyser, Support for Geyser, and Unsupported. Based on the description, please move your host entry to the "support for geyser" part & insert it to keep alphabetical order.

Reply:
Thanks for the clarification, it's already solved.